terraform-provider-constellation/BUILD.bazel

```diff
@@ -10,6 +10,22 @@ go_binary(
     visibility = ["//visibility:public"],
 )
 
+[
+    go_cross_binary(
```

Comment:
Can you add this build target to `devbuild` so we build the provider binary with everything else?
Should also make manual testing a lot easier

Reply:
We do already: https://github.com/edgelesssys/constellation/blob/ac056ae01082aa3f6bbb9ef57d58ff5731359bbd/bazel/devbuild/BUILD.bazel#L23